src/backend/TrafficCourts/Citizen.Service/Mappings/MappingProfile.cs

```diff
@@ -9,11 +9,25 @@ public class NoticeOfDisputeToMessageContractMappingProfile : Profile
     public NoticeOfDisputeToMessageContractMappingProfile()
     {
         CreateMap<NoticeOfDispute, SubmitNoticeOfDispute>()
-        .ForMember(dest => dest.DisputeCounts, opt => opt.MapFrom(src => src.DisputeCounts));
+            .ForMember(dest => dest.DisputeCounts, opt => opt.MapFrom(src => src.DisputeCounts));
```

Comment:
Do you have unit tests for these mappings?

Reply:
I think these one is here for a long time, I just add a tab in front of it